HW9/HW9.cpp

```diff
@@ -0,0 +1,176 @@
+﻿#include <iostream>
+
+void task1();
+void task2();
+void task3();
+void task4();
+void task5();
+
+int main()
+{
+    task1();
+	task2();
+	task3();
+	task4();
+	task5();
+}
+
+void translateArray(int numbers[], int size)
+{
+	for (int i = 0; i < size; i++)
+	{
+		if (numbers[i] > 0)
+		{
+			numbers[i] *= 2;
+		}
+		else if (numbers[i] < 0)
+		{
+			numbers[i] = 0;
+		}
+
+		std::cout << numbers[i] << std::endl;
+	}
+}
+
+void task1()
+{
+	constexpr int size = 10;
+	int numbers[size] = {};
+
+	for (int i = 0; i < size; i++)
+	{
+		std::cout << "Enter the number (item " << i + 1 << "): ";
+		std::cin >> numbers[i];
+	}
+
+	translateArray(numbers, size); 
+}
+
+void toUppercase(char str[]) {
+	
+	int i = 0;
+	while (str[i] != '\0') {
+		if (str[i] >= 'a' && str[i] <= 'z') {
+			str[i] = str[i] - 'a' + 'A';
```

Comment:
calculate this once 'a' + 'A' outside of the loop to not repeat it every iteration of the loop